src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -299,6 +299,25 @@ def get_request_status(
             detail="Cannot specify both succeeded and failed query params.",
         )
 
+    for end, start, field_name in [
+        [created_lt, created_gt, "created"],
+        [completed_lt, completed_gt, "completed"],
+        [errored_lt, errored_gt, "errorer"],
+        [started_lt, started_gt, "started"],
+    ]:
+        if end is None or start is None:
+            continue
+        if not (isinstance(end, datetime) and isinstance(start, datetime)):
+            continue
+        else:
```

Comment:
do we actually need an else here? Since we `continue` in the above conditions

Reply:
We don't, I did it to keep the diff smaller since the following lines were already at this indentation level